dpnp/dpnp_iface_trigonometric.py

```diff
@@ -395,9 +395,9 @@ def cos(x1):
 
     x1_desc = dpnp.get_dpnp_descriptor(x1)
     if x1_desc:
-        return dpnp_cos(x1_desc)
+        return dpnp_cos(x1_desc, out=out)
```

Comment:
```suggestion
        return dpnp_cos(x1_desc, out)
```
It looks like we use positional parameters and do not use "named" (keyword) parameters in Cython functions.